packages/vscode-extension/src/debug/depsChecker/taskChecker.ts

```diff
@@ -82,9 +82,9 @@ function addCheckResultsForTelemetry(
   properties: { [key: string]: string },
   errorProps: string[]
 ): void {
-  const [resultRaw, resultSafe] = convertCheckResultsForTelemetry(checkResults);
-  properties[TelemetryProperty.DebugCheckResultsSafe] = resultSafe;
-  properties[TelemetryProperty.DebugCheckResults] = resultRaw;
+  // const [resultRaw, resultSafe] = convertCheckResultsForTelemetry(checkResults);
```

Comment:
What's this "comment out" for?

Reply:
We will use different error code and properties to identify the check result instead of a json string. For example, there is a new error named: PortsConflictError with "ports" and "occupied-ports" properties to identify the port numbers to check and the ones that occupied when local debug. So these two properties is no longer needed.